src/lib/beta-features.ts

```diff
@@ -0,0 +1,53 @@
+import { lockAppdata, unlockAppdata, loadUserData, saveUserData } from 'src/storage/user-data';
+
+export interface BetaFeatureDefinition {
+	label: string;
+	key: string;
+	default: boolean;
+	description?: string;
+}
+
+export const BETA_FEATURES_DEFINITION: Record< keyof BetaFeatures, BetaFeatureDefinition > = {
```

Comment:
It seems it's not used outside
```suggestion
const BETA_FEATURES_DEFINITION: Record< keyof BetaFeatures, BetaFeatureDefinition > = {
```

Reply:
Thanks, removed in d4e7aaa3b4b7dbac011a2671bd898a94bd70a16a.